tests/e2e/cluster-life-cycle/kind_install_delete/test_kind.py

```
@@ -0,0 +1,91 @@
+import pytest
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)